src/test/java/com/rultor/agents/MailsTest.java

```diff
@@ -0,0 +1,147 @@
+ * modification, are permitted provided that the following conditions
+ * are met: 1) Redistributions of source code must retain the above
+ * copyright notice, this list of conditions and the following
+ * disclaimer. 2) Redistributions in binary form must reproduce the above
+ * copyright notice, this list of conditions and the following
+ * disclaimer in the documentation and/or other materials provided
+ * with the distribution. 3) Neither the name of the rultor.com nor
+ * the names of its contributors may be used to endorse or promote
+ * products derived from this software without specific prior written
+ * permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT
+ * NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+ * FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL
+ * THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+ * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+ * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+ * OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.rultor.agents;
+
+import com.jcabi.email.Envelope;
+import com.jcabi.email.Postman;
+import com.jcabi.manifests.Manifests;
+import com.jcabi.xml.XMLDocument;
+import com.rultor.spi.Agent;
+import com.rultor.spi.Profile;
+import com.rultor.spi.Talk;
+import java.io.IOException;
+import org.apache.commons.lang3.StringUtils;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mockito;
+import org.xembly.Directives;
+
+/**
+ * Tests for ${@link com.rultor.agents.Mails}.
+ *
+ * @author Yuriy Alevohin (alevohin@mail.ru)
+ * @version $Id$
+ * @since 2.0
+ */
+public final class MailsTest {
+
+    /**
+     * Mails can send a mail.
+     * @throws Exception In case of error.
+     */
+    @Test
+    @Ignore
+    public void sendsMail() throws Exception {
+        final Postman postman = Mockito.spy(Postman.CONSOLE);
+        final Profile profile = this.profile();
+        final Agent agent = new Mails(
+            profile,
+            postman
+        );
+        final Talk talk = MailsTest.talk();
+        agent.execute(talk);
+        final ArgumentCaptor<Envelope> captor =
+            ArgumentCaptor.forClass(Envelope.class);
+        Mockito.verify(postman).send(captor.capture());
+        final Envelope envelope = captor.getValue();
+        MatcherAssert.assertThat(
+            envelope.unwrap().getContent().toString(),
+            Matchers.allOf(
```

Comment:
you should be more precise here.
try to use ```javax.mail.Message``` methods to obtain necessary fields. this would help to the implementor to know where to put all mail attributes